airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -357,12 +398,15 @@ async def _run(self, built_connector: Container) -> StepResult:
 
 class UploadSbom(Step):
     context: PublishConnectorContext
-    title = "Upload SBOM to metadata service bucket"
     SBOM_KEY_PREFIX = "sbom"
     SYFT_DOCKER_IMAGE = "anchore/syft:v1.6.0"
     SBOM_FORMAT = "spdx-json"
     IN_CONTAINER_SBOM_PATH = "sbom.json"
     SBOM_EXTENSION = "spdx.json"
+    
+    @property
```

Comment:
This and all other `title` wrapped in `@property`: let's not do that in this PR. I know just setting `title = "something"` does not agree with mypy, but that's what we did everywhere, so it's fine. We will fix this in another pull request.